### PR TITLE
Setup, contexts stage: fixes, simplifications, improvements

### PR DESCRIPTION
**Pull request by mishanthrop**
### 1 Improved adaptive markup

Before:

2_before

After:

2_after

### 2 The already huge form has been simplified

Before:

1_before

After (ignore the second circle):

1_after

### 3 Bug is fixed

How to reproduce:
- Enter text (ok)
- Uncheck checkbox (ok)
- Enter text (ok)
- Checkbox is not checked automatically (bad)

### 4 Simplified js for context stage in setup
There was a lot of boilerplate.
Also removed duplicate code from template (`onchange` and `onclick`).

### 5 Unused code removed
Removed processing of unused data from `$_POST`:
* `web_path_auto`
* `web_url_auto`
* `mgr_path_auto`
* `mgr_url_auto`

### CODE REVIEW

File: _build/templates/default/sass/installer.scss [hunk 1]
```diff
@@ -766,42 +766,55 @@ header {
                 margin-left: 10px;
             }
         }
+    }
 
-        .col-1,
-        .col-2,
-        .col-3 {
-            line-height: 30px;
+    .context-property {
```
Comment (mishanthrop) [inline _build/templates/default/sass/installer.scss:771]: it's more readable than`.col-1`, `.col-2` and `.col-3`, isn't it?

File: setup/controllers/contexts.php [hunk 2]
```diff
@@ -34,17 +34,11 @@
 
     $settings = [];
     $settings['core_path'] = MODX_CORE_PATH;
-    $settings['web_path_auto'] = isset ($_POST['context_web_path_toggle']) && $_POST['context_web_path_toggle'] ? 1 : 0;
```
Comment (mishanthrop) [inline setup/controllers/contexts.php:37]: Nowhere did I find any mention of these variables (`web_path_auto`, `web_url_auto` etc) except in the files of this Pull Request.

File: setup/assets/js/sections/contexts.js [hunk 3]
```diff
@@ -4,67 +4,35 @@ Ext.onReady(function() {
 });
 
 MODx.ctx = function() {
-    
     return {
         init: function() {
-            Ext.get('context_web_path').on('keydown',function() {
-                Ext.get('context_web_path_toggle').set({ checked: true });
```
Comment (mishanthrop) [inline setup/assets/js/sections/contexts.js:11]: Buggy line.
The vanilla javascript for such tasks has always been more readable, predictable and portable.

**Review by Ibochkarev — APPROVED** [review]
изображение

**Review by theboxer — APPROVED** [review]

**Comment by theboxer** [comment]
@mishantrop good job, thanks for your contributions

**Review by opengeek — APPROVED** [review]

**Pull request merged**